frontend/app/route-helpers/status-route-helpers.server.ts

```diff
@@ -144,5 +143,5 @@ interface GetStatusResultUrlArgs {
 }
 
 export function getStatusResultUrl({ id, params }: GetStatusResultUrlArgs) {
-  return getPathById('$lang/_public/status/result', params) + `?id=${id}`;
+  return getPathById('public/status/result', params) + `?id=${id}`;
```

Comment:
You've copy pasted code from `main` that will surely not work in `v3.4.0`. Only apply the redirect fix. Don't change anything else because it might broke `v3.4.0`.

Reply:
Noticed it as soon as I created the PR... My bad

Reply:
Please squash the commits.